### PR TITLE
My attempt at using SQLServer to make flashcards console app

### PR DESCRIPTION
**Pull request by StanimalTheMan**
I enforce ON DELETE CASCADE via SQL Server Management Studio, not in the code.  Feel free to let me know if that's a problem.  Because I enforce in the studio, deleting a stack deletes all the related flashcards and study sessions.

### CODE REVIEW

**Comment by StanimalTheMan** [comment]
Will merge master and maybe I should do a CREATEiIF NOT EXIST to create table with delete constraints for review testing runs

**Comment by TheCSharpAcademy** [comment]
@StanimalTheMan Looking good, I'll approve it. But yeah it would have been better to add the on delete constraints to the code, as deleting a stack didn't work for me. 🥺

Other than that, well done! 👏🏼👏🏼

File: Flashcards.StanimalTheMan/Flashcards.StanimalTheMan/DatabaseHelper.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+);        
+END
+        ";
+
+        string createFlashcardsQuery = @"
+IF NOT EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'Flashcards')
+BEGIN
+    CREATE TABLE Flashcards (
+        FlashcardId INT PRIMARY KEY IDENTITY(1,1),
+        StackId INT FOREIGN KEY REFERENCES Stacks(StackId),
```
Comment (StanimalTheMan) [inline Flashcards.StanimalTheMan/Flashcards.StanimalTheMan/DatabaseHelper.cs:43]: `StackId INT FOREIGN KEY REFERENCES Stacks(StackId),` -> `StackId INT FOREIGN KEY REFERENCES Stacks(StackId) ON DELETE CASCADE,`

File: Flashcards.StanimalTheMan/Flashcards.StanimalTheMan/DatabaseHelper.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+END
+
+";
+
+        string createStudyQuery = @"
+IF NOT EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'Study')
+BEGIN
+    CREATE TABLE Study (
+        StudySessionId INT PRIMARY KEY IDENTITY(1,1),
+        StackId INT FOREIGN KEY REFERENCES Stacks(StackId),
```
Comment (StanimalTheMan) [inline Flashcards.StanimalTheMan/Flashcards.StanimalTheMan/DatabaseHelper.cs:56]: `StackId INT FOREIGN KEY REFERENCES Stacks(StackId)` -> `StackId INT FOREIGN KEY REFERENCES Stacks(StackId) ON DELETE CASCADE`

**Pull request merged**